### DISCUSSION TITLE
Can Shlink work on OpenLitespeed?  (lsphp)

### DISCUSSION THREAD

**Discussion by morphinestyle** [shlinkio/shlink#1949] [Help wanted] [answered]
On my server I have an installation with Cyberpanel that uses Open Litespeed, has anyone managed to make Shlink work with this same configuration?

**Comment by acelaya** [comment, marked as answer]
Are you having any specific problem? If so, can you share more information?

If you are just asking, I remember interacting with people in issues and/or discussions that were using litespeed, so I would say it's possible.

**Comment by morphinestyle** [comment]
Hello Alejandro!
Yes, i need help please! When the installation is complete I navigate to the main domain and get:
```
404
Page not found.
The page you requested could not be found.
```
I'm using distro version, on PHP 8.2 and MariaDB 10.11 on Ubuntu 22.04.

Even using Apache as a proxy server on Litespeed I get the same error.

What am I doing wrong?

**Comment by morphinestyle** [comment]
I apologize.
I didn't realize the web client was separate.
